components/brave_rewards/browser/database/database_contribution_queue.h

```diff
@@ -35,10 +31,14 @@ class DatabaseContributionQueue: public DatabaseTable {
   bool DeleteAllRecords(sql::Database* db);
 
  private:
+  bool CreateTableV9(sql::Database* db);
+
+  bool MigrateToV9(sql::Database* db);
+
+  bool MigrateToV15(sql::Database* db);
+
```

Comment:
nit: observation - as all Create/Migrate methods operate on `sql::Database` all such methods should be const? (As a practice across the board I mean - there's no need to change this PR. But, if agreed we can address in another ticket).

Reply:
sure you can add this to your ticket

Reply:
Done :+1: